yarn.lock

```diff
@@ -1439,6 +1439,24 @@
   resolved "https://registry.yarnpkg.com/@fluentui-react-native/design-tokens-windows/-/design-tokens-windows-0.13.0.tgz#1d8610634920bd5decaca669cae8183c0713d2ef"
   integrity sha512-o9ir+tpie+Fm9zO+nCbJLBfzVjUhugRWRsPS8n6wBaujC1qyeUYjhoGg6ejXzKc0nVCkIsFmRRRgC0brfrPIzw==
 
+"@fluentui-react-native/framework@0.7.30":
```

Comment:
This shouldn't be in our yarn.lock. I think it's likely that your package is referring to an older version of the framework package.